.circleci/config.yml

```diff
@@ -610,8 +627,11 @@ jobs:
     resource_class: << pipeline.parameters.twoxlarge >>
     steps:
       - run_test:
+          timeout: 30m
+          # test_vm_execute_and_finalize can take over 10 minutes in the current setup
+          no_output_timeout: 20m
           workspace_member: snarkvm-synthesizer
-          flags: --test '*' --features test -- --test-threads=4
+          flags: --test '*' --features test
```

Comment:
This test runs out of memory sometimes, which is why the thread limit was put in place. It is not perfect, but it seemed to do the job without having to refactor that whole test suite.

The same is true for other tests which had a limited number of threads allocated.